### PR TITLE
Fixes #553 - Don't load Form data

### PR DESCRIPTION
**Pull request by Leashys**
adds an extension so that you can opt to clear out form data on a given task

### CODE REVIEW

**Comment by cullerton** [comment]
This should have a test.

**Comment by Leashys** [comment]
ok i wrote a test

**Comment by Leashys** [comment]
@danfunk  this is ready to merge in now BTW well if you like that test it is

File: crc/services/workflow_service.py [hunk 1]
```diff
@@ -734,6 +734,12 @@ def spiff_task_to_api_task(spiff_task, add_docs_and_forms=False):
                 # Otherwise, just use the curreent title.
         elif task.title and ' ' in task.title:
             task.title = task.title.partition(' ')[2]
+
+        if task.properties and "clear_data" in task.properties:
```
Comment (danfunk) [inline crc/services/workflow_service.py:738]: @Leashys - this won't work for Alex's case that he described this morning.  
What we need to do is /prevent/ the previously submitted form data from overwriting what he has set in a previous step.

Comment (danfunk) [reply crc/services/workflow_service.py:738]: a good test for this is
Task 1:  Have a radio button for "a", "b", "c"
Task 2: Display a text box that contains the value selected on the previous page.
Task 3: End Task

It should be possible to change the value in Task 1, and have that change reflected in task 2.

**Review by danfunk — CHANGES_REQUESTED** [review]

**Comment by Leashys** [comment]
@danfunk @danfunk @danfunk

**Pull request merged**